src/content/translations/pt-br/security/index.md

```diff
@@ -0,0 +1,287 @@
+
+Um erro comum que os indivíduos cometem é usar uma combinação de duas a três palavras comuns do dicionário relacionadas. Senhas como essa são inseguras porque são propensas a uma simples técnica de usurpação conhecida como [ataque de dicionário](https://wikipedia.org/wiki/Dictionary_attack).
+
+```md
+Exemplo de uma senha fraca: CuteFluffyKittens!
+
+Exemplo de uma senha forte: ymv\*azu.EAC8eyp8umf
+```
+
+Outro erro comum é o uso de senhas que podem ser facilmente adivinhadas ou descobertas por meio de [engenharia social](<https://wikipedia.org/wiki/Social_engineering_(security)>). Incluir o nome de solteira de sua mãe, os nomes de seus filhos ou animais de estimação ou datas de nascimento em sua senha não é seguro e aumentará o risco de sua senha ser invadida.
+
+#### Práticas recomendadas relacionadas a senhas: {#good-password-practices}
+
+- Crie senhas tão extensas quanto permitidas pelo gerador de senhas ou pelo formulário que você está preenchendo
+- Use uma combinação de maiúsculas, minúsculas, números e símbolos
+- Não use detalhes pessoais, como nomes de membros da família, em sua senha
+- Evite palavras comuns do dicionário
+
+[Mais sobre a criação de senhas fortes](https://terranovasecurity.com/how-to-create-a-strong-password-in-7-easy-steps/)
+
+### Use senhas exclusivas para tudo {#use-unique-passwords}
+
+Uma senha forte não fornece muita proteção se a senha for revelada em uma violação de dados. O site[Have I Been Pwned](https://haveibeenpwned.com)permite verificar se as suas contas foram envolvidas em quaisquer violações de dados armazenadas na base de dados deles. Caso sim, **você deveria alterar as senhas "pwned" imediatamente**. Usar senhas únicas para cada conta diminui o risco de os hackers obterem acesso a todas as suas contas quando uma delas for comprometida.
+
+### Use um gerenciador de senhas {#use-password-manager}
+
+<InfoBanner emoji=":bulb:">
+  <div>
+    Usar um gerenciador de senhas faz com que ele crie senhas fortes, únicas e que sejam lembradas! Nós <strong>fortemente</strong> recomendamos usar um, e a maioria deles é gratuita!
+  </div>
+</InfoBanner>
+
+Lembrar de senhas fortes e exclusivas para cada conta que você possui não é o ideal. Um gerenciador de senhas oferece um cofre seguro e criptografado para todas as suas senhas que você pode acessar por meio de uma senha mestra forte. Eles também sugerem senhas fortes quando você se inscreve em um novo serviço, para que você não precise criar as suas próprias. Muitos gerenciadores de senhas também irão informar se você esteve envolvido em uma violação de dados, permitindo que você altere as senhas antes de qualquer ataque malicioso.
+
+![Exemplo do uso de um gerenciador de senhas](./passwordManager.png)
```

Comment:
These images (using the English path) are broken :/

Reply:
This appears to be fixed after merging in Pablo's plugin changes